core/chains/evm/logpoller/orm.go

```diff
@@ -439,6 +453,27 @@ func validateTopicIndex(index int) error {
 	return nil
 }
 
+func (o *ORM) SelectIndexedLogsCreatedAfter(address common.Address, eventSig common.Hash, topicIndex int, topicValues []common.Hash, after time.Time, qopts ...pg.QOpt) ([]Log, error) {
```

Comment:
@connorwstein shouldn't we have here an additional argument for # of confirmations?